client/src/modules/templates/bhIprScale.tmpl.html

```diff
@@ -1,5 +1,5 @@
 <div class="btn-group" uib-dropdown dropdown-append-to-body>
-  <a class="btn btn-default" uib-dropdown-toggle>
+  <a id="ipr_scale" class="btn btn-default" uib-dropdown-toggle>
```

Comment:
This won't solve the issue, since this also isn't unique.  Every dropdown will have `id="ipr_scale"`.  Also IDs are supposed to be unique in HTML.

Can you use something from the `row` object? Like `{{row.entity.id}}`?  Whatever the column is that uniquely identifies the IPR Tax Scale in the database should be used here to select this particular row from the grid.

Alternatively, if there is a unique text (like `display_name` or `label` or something) that would work too.  It just needs to uniquely identify this row.